SignalModelInterpolation/create_signal_ws_new_cat_2d_res_bkg.py

```diff
@@ -68,34 +69,47 @@ def makeWorkspace(models, systematicss, year, cat, workspace_output, mgg_range,
 
   if doSyst:
     systematics = systematicss[year][cat]
-
     #creates splines for const values
-    const_sys_names = [name for name in systematics[masses[0]].keys() if "const" in name]
+    const_sys_names = ["fnuf_mean","fnuf_rate","fnuf_sigma","material_mean","material_rate","material_sigma","MCScale_scale_mean","MCScale_scale_rate","MCScale_scale_sigma","MCSmear_smear_mean","MCSmear_smear_rate","MCSmear_smear_sigma"]
```

Comment:
Why not use something similar to https://github.com/cmstas/flashggFinalFit/blob/3927cb1433a632b496f7494e04707fa08d95f2e7/SignalModelInterpolation/create_signal_ws_new_cat_2d.py#L99?
Having to explicitly mention uncertianties, once for "mean", "sigma" and "rate" can be really cumbersome...

Reply:
Thanks for explaining, let's not dwell too much on this. Leave it as is but add a comment to explain the situation as you do here.

Reply:
Added an explanation